composer.lock

```diff
@@ -4,24 +4,24 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "d59e38d08c48e1a49e11c9de95d6d9ed",
+    "content-hash": "a936e737473b60adca280b2a50fecda7",
     "packages": [
         {
             "name": "apimatic/core-interfaces",
-            "version": "0.1.0",
+            "version": "dev-3-php-8.2-support",
```

Comment:
run composer update locally after releasing core-interfaces, to update composer lock file in this PR

Reply:
PHP 8.2 update branch is not merged in core-interfaces-php. Composer won't resolve.